content/events/angular-workshop.mdx

```diff
@@ -8,7 +8,15 @@ eventHeader:
   heroBackground: /images/background/polygonBackground.png
   altText: 'Angular Workshop '
   imgOverlay: /images/background/Angular-Workshop.png
-footer: ''
+title: <span class="text-sswRed">Angular Hands on 2-day Workshop</span>
```

Comment:
These <span> nullify the h1 tag 

1. Please fix! (style the h1 on these pages SSW Red without <span>)

Reply:
fixed, thanks